tools/tensorflow-quantization/README.md

```diff
@@ -1,12 +1,12 @@
 # <span style="color:green"> **NVIDIA TensorFlow 2.x Quantization** </span>
 
 This TensorFlow 2.x Quantization toolkit quantizes (inserts Q/DQ nodes) TensorFlow 2.x Keras models for Quantization-Aware Training (QAT).
-We follow NVIDIA's QAT recipe, which leads to optimal model acceleration with [TensorRT](https://docs.nvidia.com/deeplearning/tensorrt/developer-guide/index.html) on NVIDIA GPUs and hardware accelerators.
+We follow NVIDIA's QAT recipe, which leads to optimal model acceleration with [TensorRT](https://docs.nvidia.com/deeplearning/tensorrt/developer-guide/index.html) on NVIDIA GPUs and hardware accelerators. 
```

Comment:
There are a few extra whitespaces added in this commit. Can you remove them?

Reply:
Done